event/event_cache_test.go

```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-var mockInterval = 10 * time.Millisecond
+var mockInterval = 40 * time.Millisecond
```

Comment:
I had a CI failure, timing here was a little tight anyway, also ugly, but pre-existing and we'll get around to that.